package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-plugin-upload-s3",
-  "version": "1.0.1",
+  "version": "1.2.1",

```

Comment:
Hopefully this will do something.  If not, I can manually release 1.2.1, because the version here ought to match the published release version.